### DISCUSSION TITLE
Laragon on Linux

### DISCUSSION THREAD

**Discussion by haseebjduk** [leokhoa/laragon#395] [Q&A]
What should I do to use Laragon on ubuntu.

**Comment by Pen-y-Fan** [comment]
Laragon only works on Windows. It should only be used for Local development.

If you have experience with [Wine](https://wiki.winehq.org/Debian), it may be worth trying. Laragon requires MS runtime c++ libraries, which may be a problem.

Search for setting up a LAMP stack on Ubuntu, there are plenty of tutorials.

**Reply by yocki24** [reply]
can i change php version on LAMP like Laragon?
